ui/desktop/app/styles/app.scss

```diff
@@ -105,12 +82,11 @@
 
   .rose-header-brand,
   .rose-header-nav,
-  .rose-header-utilities .rose-dropdown,
+  .rose-header-utilities,
```

Comment:
This css now does something different, it was previously a nested selector for `.rose-dropdown` and now is targeting the entire `.rose-header-utilities`

Reply:
Oh good catch